test/e2e/tests/metrics/dapp-viewed.spec.js

```diff
@@ -75,7 +75,7 @@ describe('Dapp viewed Event @no-mmi', function () {
         dapp: true,
         fixtures: new FixtureBuilder()
           .withMetaMetricsController({
-            metaMetricsId: 'fake-metrics-id',
+            metaMetricsId: 'fake-metrics-id-fd20', // 1% sample rate for dapp viewed event
```

Comment:
Nit: can we extract this as a variable at the top of test file to avoid duplication o`'fake-metrics-id-fd20'`?